pandas/core/frame.py

```diff
@@ -4354,6 +4354,9 @@ def duplicated(self, subset=None, keep='first'):
         from pandas.core.sorting import get_group_index
         from pandas._libs.hashtable import duplicated_int64, _SIZE_HINT_LIMIT
 
+        if self.columns.empty:
+            return Series()
```

Comment:
``dtype='bool'``?

Reply:
Good to be explicit here. We'll be changing this from `float` to `object` soon.